lib/utils/angular/properties/properties.dart

```diff
@@ -68,12 +51,39 @@ bool getBool(inputValue) {
       inputValue, 'inputValue', 'Expected a String, or bool type');
 }
 
+/// Parses html attribute [String] to a [bool].
+///
+/// Should be used to parse values passed to @Attribute constructor argument.
+///
+/// This does not fully follow the HTML boolean attribute definition
+/// (https://stackoverflow.com/a/4139805), as 'false' String will be parsed
+/// to false value.
+///
+/// When no attribute is present [defaultValue] value is returned (by default
+/// false).
```

Comment:
Nit: a default of the default :) ... I would keep this DRY and drop the "(by default false)" since this can be reckoned by reading the function signature.